nodes/widgets/locales/en-US/ui_template.html

```diff
@@ -11,58 +11,49 @@
 
 ### Type: Widget
 When the scope is set to **Widget**, the template will be rendered in the Dashboard as HTML.
-You can render dynamic content using any VueJS data-binding expressions, (e.g. `v-if`, `v-for`)
-and access incoming data to the node via the `msg` object, e.g:
-
-<pre>
-&lt;template&gt;
-  &lt;div&gt;
-      &lt;h2&gt;Counter&lt;/h2&gt;
-      &lt;p&gt;Current Count: {{ count }}&lt;/p&gt;
-      &lt;p class="test-class"&gt;Formatted Count: {{ formattedCount }}&lt;/p&gt;
-      &lt;v-btn @click="increase()"&gt;Increment&lt;/v-btn&gt;
-      &lt;/div&gt;
-&lt;/template&gt;
-
-&lt;script&gt;
-  export default {
-      data() {
-          // define variables available component-wide
-          // (in &lt;template&gt; and component functions)
-          return {
-              count: 0
-          }
-      },
-      watch: {
-          // watch for any changes of "count"
-          count() {
-              if (this.count % 5 === 0) {
-                  this.send({payload: 'Multiple of 5'})
-              }
-          }
-      },
-      computed: {
-          // automatically compute this variable
-          // whenever VueJS deems appropriate
-          formattedCount() {
-              return `${this.count} Apples`
-          }
-      },
-      methods: {
-          // expose a method to our &lt;template&gt; and Vue Application
-          increase() {
-              this.count++
-          }
-      }
-  }
-&lt;/script&gt;
-
-&lt;style&gt;
-.test-class {
-  color: red;
-}
-&lt;/style&gt;
-</pre>
+
+You can render dynamic content using any VueJS data-binding expressions, (e.g. `v-if`, `v-for`, `:class`)
+and access incoming data to the node via the `msg` object.
+
+#### Working with Variables
+
+Any variables that you want to render into your `<template />` are done so in one of two ways:
+
+- **Attribute Binding** - Use `:` to bind a variable to an attribute. Anything inside the `""` here is treated as JavaScript, for example: 
+
+```html
+&lt;p :class="msg.payload">Hello World&lt;/p>
```

Comment:
```suggestion
\<p :class="msg.payload">Hello World\</p>
```

Joe, these can all be escaped simply with a `\`

Do you mind if I edit the file and replace them all in one go instead of individual suggestions?

Reply:
> Do you mind if I edit the file and replace them all in one go instead of individual suggestions?

Please do

Reply:
bah, unfortunately the markdown processor used in NR doesnt correctly use `\` as an escape 😢 (it should)